packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.routing.js

```diff
@@ -42,6 +43,13 @@ export default /* @ngInject */ ($stateProvider) => {
           OvhApiCloud => OvhApiCloud.v6().schema().$promise
             .then(schema => get(schema, VERSION_ENUM_KEY)),
 
+        highestVersion: /* @ngInject */
+          versions => reduce(versions, (currentHighestVersion, version) => (
```

Comment:
Does it make sense to use `_.max` instead ? https://lodash.com/docs/3.10.1#max

Reply:
Thats an awesome idea. This is changed. Thanks!